src/Containers/MovieDetailsContainer/MovieDetailsContainer.jsx

```diff
@@ -0,0 +1,17 @@
+import { Link, useLoaderData, useSearchParams } from 'react-router-dom';
+import { MovieDetails } from '../../Components/MovieDetails/MovieDetails';
+import styles from './MovieDetailsContainer.module.css';
+
+export const MovieDetailsContainer = () => {
+  const movie = useLoaderData();
```

Comment:
You can move `useLoaderData();` to `MovieDetails` component.
`movie` is not needed here.